shell/platform/embedder/embedder.h

```diff
@@ -1751,13 +1769,22 @@ typedef bool (*FlutterLayersPresentCallback)(const FlutterLayer** layers,
                                              size_t layers_count,
                                              void* user_data);
 
+/// The callback invoked when the embedder should present
+/// to a view.
```

Comment:
nit: does this actually hit 80 columns?

Reply:
Nice catch! I had wrapped this manually for the design doc but forgot to update it here - fixed!